src/components/docs/Breadcrumbs.tsx

```diff
@@ -1,27 +1,25 @@
-import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from '@chakra-ui/react';
+import { Breadcrumb, BreadcrumbItem, Link } from '@chakra-ui/react';
+import NextLink from 'next/link';
 import { FC } from 'react';
 
 interface Props {
-  router: any
+  router: any;
 }
 
 export const Breadcrumbs: FC<Props> = ({ router }) => {
-  let pathSplit = router.asPath.split('/')
-  pathSplit = pathSplit.splice(1,pathSplit.length)
+  let pathSplit = router.asPath.split('/');
+  pathSplit = pathSplit.splice(1, pathSplit.length);
 
   return (
     <Breadcrumb mb={10}>
       {pathSplit.map((path: string, idx: number) => {
         return (
           <BreadcrumbItem key={path}>
-            <BreadcrumbLink
-              color={ idx+1 === pathSplit.length ? 'body' : 'primary' }
-              href={`/${pathSplit.slice(0,idx+1).join('/')}`}
-            >
-              {path}
-            </BreadcrumbLink>
+            <NextLink href={`/${pathSplit.slice(0, idx + 1).join('/')}`} passHref>
```

Comment:
I meant to wrap `BreadcrumbLink` (`BreadcrumbLink`is more semantically correct here vs  Chakra's`Link`), does this work?

```
<NextLink href={`/${pathSplit.slice(0, idx + 1).join('/')}`} passHref>
  <BreadcrumbLink color={ idx+1 === pathSplit.length ? 'body' : 'primary' }>
    {path}
  </BreadcrumbLink>
</NextLink>
```

Reply:
Ah good point